Readme.md

```diff
@@ -2,27 +2,34 @@
 
 ## About
 
-When it comes to modernizing the applications, migrating to Mapstruct from Dozer mapper is a primary part for most of the Java enterprise applications that are running sucessfully for more than a decade.
+When it comes to modernizing the applications, migrating to Mapstruct from Dozer mapper is a primary part for most of the Java enterprise applications that are running successfully for more than a decade.
 
 This Intellij plugin will help us to do the migration faster by generating mapstruct code for the dozer xml.
 
 ## Steps to add this plugin to ur ide
```

Comment:
Why we have heading when there is no content?

Reply:
BTW, we can mention under this heading that they can install the plugin from Marketplace. And mention the compatible version.

Reply:
Fixed.